_includes/contact-form.html

```diff
@@ -53,6 +53,7 @@
         <input name="page_name" type="hidden" value="{{ page.title | liquify }}">
         <input name="page_uri" type="hidden" value="{{ school.url }}{{ site.baseurl }}{{ page.url }}">
         <input name="hutk" type="hidden" value="">
+        <input name="redirect_url" type="hidden" value="https://austincodingacademy.com/thank-you/">
```

Comment:
This should probably be interpolated for lubbock and ttu `{{ school.url }}/thank-you/`

Reply:
and the ones below